pages/content/amp-dev/documentation/guides-and-tutorials/contribute/vendor-contributions/release-schedule.html

```diff
@@ -0,0 +1,284 @@
+---
+$title: Release Schedule
+---
+{% do doc.styles.addCssFile('/css/components/organisms/release-schedule.css') %}
+{% do doc.amp_dependencies.add('amp-script', '0.1') %}
+
+<p>For a detailed description of the various release channels, see <a href="https://github.com/ampproject/amphtml/blob/master/contributing/release-schedule.md">release-schedule.md</a>. There is also a <a href="./release-schedule-accessible.html">screen-reader accessible</a> version of this page.</p>
+
+<amp-script layout="intrinsic" width="1050" height="0" script="calendar-script">
```

Comment:
Curious: why `height=0`?

Reply:
Annoying workaround - `<amp-script>` requires a `height` attribute but `layout="intrinsic"` ignores the height